systemtests/src/main/java/com/github/streamshub/systemtests/constants/Constants.java

```diff
@@ -1,14 +1,18 @@
 package com.github.streamshub.systemtests.constants;
 
-import com.github.streamshub.console.api.v1alpha1.Console;
-import io.fabric8.kubernetes.api.model.HasMetadata;
-
 public class Constants {
     private Constants() {}
 
     // --------------------------------
     // ------------ General -----------
     // --------------------------------
+    /**
+     * Cluster
+     */
+    public static final String KUBE_DNS = "192.168.49.2.nip.io";
```

Comment:
Is this IP fixed?

Reply:
I believe for minikube it is, shall i add an ENV for it to make it more flexible?